security_scheme.go

```diff
@@ -0,0 +1,89 @@
+package spec3
+
+import (
+	"encoding/json"
+
+	"github.com/go-openapi/spec"
+	"github.com/go-openapi/swag"
+)
+
+// SecurityScheme defines reusable Security Scheme Object
+// as described in https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#securitySchemeObject
+type SecurityScheme struct {
```

Comment:
we can't reuse `SecurityScheme` model from go-openapi, amiright? i see in v2 spec, there's a field named `flow` but in v3, the field renamed to `flows` so it's not backward-compatible?

Reply:
yes, compare https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#securitySchemeObject with https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#securityDefinitionsObject

`flows` in `v3` are defined as objects as opposed to a string value that we have in `v2`